lib/Elastica/Util.php

```diff
@@ -197,6 +197,6 @@ public static function convertRequestToCurlCommand(Request $request)
      */
     public static function debugEnabled()
     {
-        return defined('DEBUG') && DEBUG;
+        return defined('ELASTICA_DEBUG') && ELASTICA_DEBUG;
```

Comment:
Though, only difference I see between "debug mode" and not "debug mode" is just two things: 
* Would `$response->setQueryTime` be invoked after request.
* Would curl option `CURLINFO_HEADER_OUT` be applied or not.

Both of them has no impact on performance, is this really worth to keep `debug` option?